css/obBase.css

```diff
@@ -202,13 +202,41 @@ form {
   width: calc(100% - 230px);
 }
 
-#pageNav {
+#appBar {
   position: fixed;
-  top: 0;
+  width: 100%;
+  z-index: 14;
+  background-color: #327EB8;
+  padding: 10px 8px;
+  box-sizing: border-box;
   -webkit-app-region: drag;
-  z-index: 13;
 }
 
+#appBar .windowControls {
+  position: absolute;
+  left: 8px;
+  top: 8px;  
+}
+
+html:not(.platform-mac) #appBar .windowControls {
+  left: auto;
+  right: 8px;
+  top: 8px;
+}
+
```

Comment:
Mac tooltips do not go off the left side because the tooltip-left class is only applied by the template if the platform is not Mac.